returnsrcvr-qual/build.gradle

```diff
@@ -14,7 +14,7 @@ sourceCompatibility = 1.8
 
 dependencies {
     // This dependency is found on compile classpath of this component and consumers.
-    implementation 'org.checkerframework:checker-qual:2.10.0'
```

Comment:
The change implementation=>api doesn't seem related to the main point of this pull request, but OK.

Reply:
Yes true.  That was for consistency with `object-construction-qual` which makes this an api dependence.